tox.ini

```diff
@@ -0,0 +1,45 @@
+# For more information about tox, see https://tox.readthedocs.io/en/latest/
+[tox]
+envlist = py{38,39,310}-{linux,macos,windows}-{pyqt,pyside}
```

Comment:
I think the OS name was only there because the numcodecs dep on Python 3.9 and macOS had to be installed from conda for some reason. I removed it now and we'll see what happens.

Also added pyqt6 and pyside6 based on your setup.